cluster/get-kube.sh

```diff
@@ -61,16 +61,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# TODO(pipejakob): Use dl.k8s.io/bazel once the redirection uses the correct bucket.
+KUBERNETES_BAZEL_RELEASE_URL="${KUBERNETES_BAZEL_RELEASE_URL:-${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release-dev/bazel}}"
```

Comment:
It's definitely not necessary, but I was following the pattern that we have for the CI release url defaulting on the next line:

> KUBERNETES_CI_RELEASE_URL="${KUBERNETES_CI_RELEASE_URL:-${KUBERNETES_RELEASE_URL:-https://dl.k8s.io/ci}}"

I was trying to keep the bazel-fetching implementation as close to possible as the existing CI fetching so there weren't any surprises for people who try to switch over, but I can also rip this out if you think it's not useful.